thenewboston_node/blockchain/serializers/blockchain_states_meta.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework_dataclasses.serializers import DataclassSerializer
+
+from thenewboston_node.business_logic.models.blockchain_state import BlockchainStateMeta
+
+
+class BlockchainStatesMetaSerializer(DataclassSerializer):
+
+    class Meta:
+        dataclass = BlockchainStateMeta
```

Comment:
let's not create data class for this to avoid mixing business logic dataclasses with representation dataclasses. Instead use DRF serializers features